### PR TITLE
Anna Lenksjö Business Siter

### PR DESCRIPTION
**Pull request by annalenksjo**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,97 @@
...
+</section>
+
+
+<!-- ACCORDION -->
+
+  <section class="accordion">
+      <h1>FAQ!</h1>
+
+         <h4 id ="accordionHeader" class="accordionHeader"> Where are you located?</h4>
+         <p id ="accordionContent" class="answer"> We are a small business, located in the Nordics.</p>
```
Comment (HIPPIEKICK) [inline code/index.html:79]: Remove the space before the `=` :)

File: code/script.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+let accordionHeader = document.getElementById("accordionHeader");
+let accordionContent = document.getElementById("accordionContent");
+
+let accordionHeaderTwo = document.getElementById("accordionHeaderTwo");
+let accordionContentTwo = document.getElementById("accordionContentTwo");
+
+let accordionHeaderThree = document.getElementById("accordionHeaderThree")
+let accordionContentThree = document.getElementById("accordionContentThree")
```
Comment (HIPPIEKICK) [inline code/script.js:8]: These can be declared with `const` becasue the values are not changing. Also, be consistent in whether you're using semicolons or not

File: code/script.js [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
+
+console.log(accordionHeader);
+console.log(accordionContent);
+console.log(accordionHeaderTwo);
```
Comment (HIPPIEKICK) [inline code/script.js:12]: Remove console.logs before production

File: code/script.js [hunk 4]
```diff
@@ -1,9 +1,24 @@
...
+ accordionContent.classList.toggle("active"); 
+})
+
+accordionHeaderTwo.addEventListener("click", () => {
+accordionContentTwo.classList.toggle("activeTwo");
+})
+
+accordionHeaderThree.addEventListener("click", () => {
+    accordionContentThree.classList.toggle("activeThree");
+    })
```
Comment (HIPPIEKICK) [inline code/script.js:24]: Remove indentation here :)

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,254 @@
...
+
+  .welcome-title {
+    font-size: 150px;
+    margin: 0px;
+  }
+
+  .welcome-sub {
+    font-size: 50px;
+    margin: 0px;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:224]: A little too wide content in tablet:
Skärmavbild 2021-02-09 kl  18 03 37

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Anna! Let's clean up a bit by fixing the indentation and removing unused code and console.logs :) Check out the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) for project specific tips!

**Comment by annalenksjo** [comment]
I have corrected most of the comments. I hope it is okay now :-) I have not implemented for example the main-section but I will take those comments with me for future projects.

**Comment by annalenksjo** [comment]
I have corrected the project so I hope it can be approved now.

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**